stl/inc/atomic

```diff
@@ -1660,6 +1852,23 @@ public:
         return this->compare_exchange_strong(_Expected, _Desired, _Combine_cas_memory_orders(_Success, _Failure));
     }
 
+#if _HAS_CXX20
+    using _Base::wait;
+    void wait(const _Ty _Expected, const memory_order _Order = memory_order_seq_cst) const volatile noexcept {
+        const_cast<atomic*>(this)->_Base::wait(_Expected, _Order);

```

Comment:
I believe the base implementation is `const`, so this `const_cast` can be minimized:
```suggestion
        const_cast<const atomic*>(this)->_Base::wait(_Expected, _Order);
```